tools/GATK-VariantRecalibrator-Indels.cwl

```diff
@@ -17,13 +17,12 @@ doc: |
             -recalFile    $tmpDir/out.recal \
             -tranchesFile $tmpDir/out.tranches \
             -rscriptFile  $tmpDir/out.R \
-            -nt 4 \
```

Comment:
Hmmmm...  Good eye @pvanheus . Wouldn't it be better if  all parallelization options specified by the user depending on their set up (maybe they like to use different options for different stages or so) . I didn't find a specific recommendation from the GATK on the VQSR stage, but they have something [here](http://gatkforums.broadinstitute.org/gatk/discussion/1975/how-can-i-use-parallelism-to-make-gatk-tools-run-faster) for the tools upstream; and whether to parallelize at the engine level or the tool level.